waterloop-site/src/pages/Contact.tsx

```diff
@@ -12,29 +12,29 @@ const Contact: React.FC = () => (
       description="We'd love to get in touch!"
       anchor="#contact-form"
     ></Hero>
+
     <div id={"#contact-form"}></div>
     <div className="pageContainer">
       <div className="contactFormContainer">
+
         <ContactForm
           title="Send Us a Message"
           desc="Want to get in touch? Submit this form or drop us a message at contact@waterloop.ca"
         ></ContactForm>
+
+        <ContactForm
+          title = "Sign Up for our Newsletter"
+          desc = "Want to sign up for our newsletter? Sign up for the form below"
+          ></ContactForm>
```

Comment:
This will need to be updated to use our new newsletter post endpoint. Reach out to William park and Evan Wong for information about how to use this